CalculatorTest.py

```diff
@@ -20,6 +20,9 @@ def test_run_operation_divide(self):
         with self.assertRaises(ZeroDivisionError):
             Calculator.run_operation(9, 0, '/')
 
+    def test_run_operation_modulo(self):
```

Comment:
Very good test case. Function name matches the other test cases

Reply:
I agree.

Reply:
> Very good test case. Function name matches the other test cases

I agree.

Reply:
agreed, as it does match the other test-cases which promotes readability!